packages/types/src/sdk/search.ts

```diff
@@ -154,6 +155,7 @@ export interface QueryJson {
   }
   resource?: {
     fields: string[]
+    aggregations?: Aggregation[]
```

Comment:
Then this is the API that takes us from the SDK layer into the SQL layer. It being a nested optional field feels really weird but it made sense to me to have it live next to `fields`.